pkg/tests/ossm/operator/clusterwide_mode_test.go

```diff
@@ -77,57 +101,70 @@ func TestClusterWideMode(t *testing.T) {
+				verifyMembersInSMMR(t, meshNamespace, notInMembersList, false)
 			})
+		})
+
+		t.NewSubTest("verify memberselector operator IN").Run(func(t test.TestHelper) {
+			t.Log("Check the use of IN in memberselector")
 
 			t.LogStep("Check the use of IN operator in member selector matchExpressions")
 			oc.ApplyString(t, meshNamespace, smmrInOperator)
 			oc.WaitSMMRReady(t, meshNamespace)
 
 			t.LogStep("Check whether the SMMR shows only one namespace as members: member-0")
+			membersList := []string{"member-0"}
+			notInMembersList := []string{"member-1", "member-2", "member-3", "member-4"}
 			retry.UntilSuccess(t, func(t test.TestHelper) {
-				shell.Execute(t,
-					fmt.Sprintf("oc -n %s get smmr default", meshNamespace),
-					require.OutputContains("1/1",
-						"one namespace are in members",
-						"expected SMMR to show 1 member namespace, but that wasn't the case"))
-				shell.Execute(t,
-					fmt.Sprintf("oc -n %s describe smmr default", meshNamespace),
-					require.OutputContains("member-0",
-						"member-0 is in members",
-						"expected SMMR to show member-0 as member namespace, but that wasn't the case"))
+				verifyMembersInSMMR(t, meshNamespace, membersList, true)
+				verifyMembersInSMMR(t, meshNamespace, notInMembersList, false)
 			})
+		})
+
+		t.NewSubTest("verify multiple memberselector").Run(func(t test.TestHelper) {
+			t.Log("Check if is possible to use multiple memberselector at the same time")
+
+			t.LogStep("Check the use of multiple selector at the same time")
+			oc.ApplyString(t, meshNamespace, smmrMultipleSelectors)
+			oc.WaitSMMRReady(t, meshNamespace)
+
+			t.LogStep("Check whether the SMMR shows only namespaces as members: member-0")
+			membersList := []string{"member-0"}
+			notInMembersList := []string{"member-1", "member-2", "member-3", "member-4"}
+			retry.UntilSuccess(t, func(t test.TestHelper) {
+				verifyMembersInSMMR(t, meshNamespace, membersList, true)
+				verifyMembersInSMMR(t, meshNamespace, notInMembersList, false)
+			})
+		})
+
+		t.NewSubTest("verify memberselector operator NOTIN").Run(func(t test.TestHelper) {
+			t.Log("Check the use of NOTIN in memberselector")
 
 			t.LogStep("Check the use of NotIn operator in member selector matchExpressions")
 			oc.ApplyString(t, meshNamespace, smmrNotInOperator)
 			oc.WaitSMMRReady(t, meshNamespace)
 
 			t.LogStep("Check whether the SMMR shows all the namespaces except: member-0")
+			membersList := []string{"member-1", "member-2", "member-3", "member-4"}
+			notInMembersList := []string{"member-0"}
 			retry.UntilSuccess(t, func(t test.TestHelper) {
-				shell.Execute(t,
-					fmt.Sprintf("oc -n %s get smmr default", meshNamespace),
-					require.OutputContains("55/55",
-						"All the namespaces are in members except member-0",
-						"expected SMMR to show 55 member namespace, but that wasn't the case"))
+				verifyMembersInSMMR(t, meshNamespace, membersList, true)
+				verifyMembersInSMMR(t, meshNamespace, notInMembersList, false)
```

Comment:
The function name is misleading when you pass in `false`. Please create two functions (e.g. `assertMembers`, `assertNonMembers` or something similar).

Reply:
Done